index.bs

```diff
@@ -670,6 +686,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
     </div>
     </li>
 
+    <li><dfn>Focus distance</dfn> is a numeric camera setting that controls the focus distance of the lens.  The setting usually represents distance in millimeters to the optimal focus distance.</li>
```

Comment:
SI unit for distance is meters, not millimeters. Also, most of the lenses have clear markings (m or ft). Android uses diopters, which is not a problem, as it can be converted easily to meters.